VMBackup/main/fsfreezer.py

```diff
@@ -111,7 +111,8 @@ def __init__(self, patching, logger, hutil):
         self.hutil = hutil
         self.safeFreezeFolderPath = "safefreeze/bin/safefreeze"
         self.isArm64Machine = False
-
+        self.file_exists = True
```

Comment:
can we re-name variable to specifically what file_exists? safeFreezeBinExists? or instead you could add a comment mentioning what file we are referring to